styleguide/guides/issues.rst

```diff
@@ -0,0 +1,172 @@
+📝 Issue Tracking
+=================
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   issues/security
+   issues/bug
+   issues/feature
+   issues/refactoring
+   issues/documentation
+
+Summary
++++++++
+Issue tracking is a great tool to track pending work and improve the overall communication within a project.
+But similar to every other tool, it needs to be used with care.
+
+    | "A fool with a tool is still a fool"
+    | -- Grady Booch
+
+This guide will try to help you to get the best out of the issue tracking for you, your teammates and the project(s)
+you are working on, by answering the following questions:
+
+* When to create an issue?
+* What makes a good issue?
+* How to maintain an issue?
+* How to maintain a backlog?
+
+When to create an issue?
+++++++++++++++++++++++++
+Let me start by saying there is no perfect and one size fits all ruleset to decide this question, otherwise there would be some code doing that for us.
+The good thing though is, humans are great in dealing with fuzzy problems. Still most people want to have something to orient themselves.
+Asking more specific questions often helps in this situation.
+
+The following question(s), helped us in the past, to get a better handle on this:
+
+* What is the cost/benefit ratio of creating an issue?
+    * cost being mainly time and effort
+    * benefit being the upsides we gain from having an issue
+
+* What is the signal to noise ratio?
+    * signal being the actual code/change bringing some benefit/value to the project
+    * noises being the overhead
+
+* Is the user (api or product) impacted by this change?
+    * Is it an "internal improvement" quality of life and/or code quality
```

Comment:
The question to myself in the past was often also: Is this change worth appearing in the changelog.....

Reply:
Good point, but this would rule out internal changes which don't necessarily appear in the changelog. At least with the current way, how our changelogs are populated.